docs/getting-started/installation.md

```diff
@@ -63,6 +63,10 @@ gremlin> :remote connect tinkerpop.server conf/remote.yaml
 Notice how it's not necessary to bind any ports in order to make this example work.
 For further reading, see the [JanusGraph Server](../operations/server.md) section as well as the [JanusGraph Docker documentation](https://github.com/JanusGraph/janusgraph-docker/blob/master/README.md).
 
+!!! warning
+    Due to a known [issue](https://github.com/JanusGraph/janusgraph-docker/issues/123), `gremlin.sh` won't work in ARM
```

Comment:
We should link from the issue to this line once this is merged so we don't forget to remove the warning once we have a fix for the issue.